pkg/apis/aks.cattle.io/v1/types.go

```diff
@@ -78,6 +78,8 @@ type AKSClusterConfigSpec struct {
 	// NetworkService CIDR is the network service cidr.
 	NetworkServiceCIDR *string `json:"serviceCidr" norman:"pointer"`
 	// NetworkDockerBridgeCIDR is the network docker bridge cidr.
+	// Setting the DockerBridgeCidr field is no longer supported,
```

Comment:
```suggestion
	// Setting the dockerBridgeCidr field is no longer supported,
```
Tiny nit to match the json name.